client/src/Movies/Movie.js

```diff
@@ -1,17 +1,19 @@
-import React, { Component } from 'react';
-import axios from 'axios';
+import React, { Component } from "react";
+import axios from "axios";
 
 export default class Movie extends Component {
   constructor(props) {
     super(props);
     this.state = {
       movie: null
     };
+    console.log(props.match.params.id);
```

Comment:
undefined, `this.props` works in class Components; `props` in functional components